run.bash

```diff
@@ -63,6 +63,25 @@ if ! install_lava "${LAVA_VERSION}"; then
 	exit 1
 fi
 
+# Install yq.
+check_status (){
+	local status=$1
+	local error_msg=$2
+	if [[ $status -ne 0 ]]; then
+		echo $error_msg
+		echo "status=${status}" >> "${GITHUB_OUTPUT}"
+		exit $status
+	fi
+}
+
+metrics=$(mktemp)
+
+sudo wget -q https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /usr/local/bin/yq
+check_status $? "error: could not install yq"
+sudo chmod +x /usr/local/bin/yq
+
+yq eval '.report.metrics = "'"$metrics"'"' -i "${config}"
```

Comment:
Wouldn't `-i` modify the config file inplace? Would that affect the following steps of the job that calls the action? For instance, a step could expect to find the metrics file in a specific location that is not longer valid.

Reply:
done 52cd7cb04607531e1e5a828cb60987c54228402a